v2/pages/adminAssessment/adminAssessment.ejs

```diff
@@ -245,6 +246,79 @@
       
       <div class="panel panel-primary">
         <div class="panel-heading">
+          <h3 class="panel-title"><%= assessment_set.abbrev %><%= assessment.number %> Stats</h3>
+        </div>
+
+        <table id="questionsTable" class="table table-condensed table-hover tablesorter">
+          <thead>
+            <tr>
+              <th>Question</th>
+              <th>Mean score</th>
+              <th class="cell-center">Discrimination</th>
+              <th class="cell-center">Attempts</th>
+              <th class="cell-center">Quintiles</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% question_stats.forEach(function(row, i) { %>
+            <tr>
+              <td><%= row.number %> <%= row.title %></td>
```

Comment:
Format like in the first panel on this page, with an extra period after the question number and the cell linked to the question page.